diff --git a/out/windowsExecutableCodeSignatureVerifier.js b/out/windowsExecutableCodeSignatureVerifier.js
index c9a579eb3a7a21504ad89e663afc0f97399ac169..3d95a38f7c3b83c3ecf135da12ca31bfd633bbfa 100644
--- a/out/windowsExecutableCodeSignatureVerifier.js
+++ b/out/windowsExecutableCodeSignatureVerifier.js
@@ -44,7 +44,7 @@ function verifySignature(publisherNames, unescapedTempUpdateFile, logger) {
         // https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file
         const tempUpdateFile = unescapedTempUpdateFile.replace(/'/g, "''");
         logger.info(`Verifying signature ${tempUpdateFile}`);
-        (0, child_process_1.execFile)(...preparePowerShellExec(`"Get-AuthenticodeSignature -LiteralPath '${tempUpdateFile}' | ConvertTo-Json -Compress"`, 20 * 1000), (error, stdout, stderr) => {
+        (0, child_process_1.execFile)(...preparePowerShellExec(`"Get-AuthenticodeSignature -LiteralPath '${tempUpdateFile}' | ConvertTo-Json -Compress"`, 300 * 1000), (error, stdout, stderr) => {
             var _a;
             try {
                 if (error != null || stderr) {
